src/applications/vaos/new-appointment/components/VAFacilityPage/VAFacilityPageV2.jsx

```diff
@@ -68,8 +66,34 @@ export default function VAFacilityPageV2() {
     singleValidVALocation,
     sortMethod,
     typeOfCare,
+    fetchRecentLocationStatus,
+    recentLocations,
   } = useSelector(state => getFacilityPageV2Info(state), shallowEqual);
 
+  const sortOptions = useMemo(
+    () => {
+      const options = [
+        {
+          value: 'distanceFromResidentialAddress',
+          label: 'By your home address',
+        },
+        {
+          value: 'distanceFromCurrentLocation',
+          label: 'By your current location',
+        },
+        { value: 'alphabetical', label: 'Alphabetically' },
+      ];
+      if (featureRecentLocationsFilter && recentLocations?.length) {
```

Comment:
nit: In the design doc I think we are supposed to make this the first option in the dropdown.